### PR TITLE
Stefan/a11y fixes

### PR DESCRIPTION
**Pull request by stefan-oconnell**
### Description

List of proposed changes:

- [a11y 27863 - Fix headers and sizes]
- [a11y - 27853: Adjust landing page order, standardize collapsable padding]
- [a11y 27866: Standardize all header sizes.]
- [a11y - 27851/27860/27850: Fix wording and figure captions.]

### CODE REVIEW

File: src/components/ApplicationNameBar.tsx [hunk 1]
```diff
@@ -9,14 +9,12 @@ const ApplicationNameBar = ({ text, href }: ApplicationNameBarProps) => {
...
       <section className="container mx-auto p-4">
-        <h2>
-          <Link
-            href={href}
-            className="font-body text-lg font-bold text-white underline md:text-[28px]"
-          >
-            {text}
-          </Link>
-        </h2>
+        <Link
```
Comment (ken-blanchard) [inline src/components/ApplicationNameBar.tsx:12]: This one was previously requested to be an h2.

#4869

File: public/locales/en/status.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "error": {
       "required": "The given name(s) is required."
     },
-    "help-message": "Enter first name(s) <strong>and any middle name(s)</strong> of the person who the passport is for.",
+    "help-message": "Enter first name(s) <strong>and any middle name(s)</strong> of the person who the passport is for. <strong>Make sure the information you enter matches your passport application.</strong>",
```
Comment (ken-blanchard) [inline public/locales/en/status.json:37]: We would also need to add these details on the 'email' page

Comment (stefan-oconnell) [reply public/locales/en/status.json:37]: Donezo.

File: src/components/ApplicationNameBar.tsx (hunk 1 above)
Comment (stefan-oconnell) [reply src/components/ApplicationNameBar.tsx]: Fixed

**Review by ken-blanchard — APPROVED** [review]

**Pull request merged**